src/module_wrap.cc

```diff
@@ -237,6 +279,17 @@ void ModuleWrap::Namespace(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(result);
 }
 
+void ModuleWrap::GetStatus(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<Object> that = args.This();
+  ModuleWrap* obj = Unwrap<ModuleWrap>(that);
+  CHECK_NE(obj, nullptr);
```

Comment:
what does ASSIGN_OR_RETURN_UNWRAP do, and why should it be used here but not anywhere else that i do CHECK_NE

Reply:
If you look at the definition of the macro you’ll see that it basically attempts to unwrap, and upon an empty wrapper (result of unwrapping is nullptr) returns immediately.

Seems like it’s just this file that doesn’t use the macro. In that case it’s fine, but in the rest of the code base we use that instead of manually unwrapping and checking the result.